contrib/platform/src/com/sun/jna/platform/win32/COM/Accessible.java

```diff
@@ -0,0 +1,85 @@
+/* Copyright (c) 2021 Mo Beigi, All Rights Reserved
+ *
+ * The contents of this file is dual-licensed under 2
+ * alternative Open Source/Free licenses: LGPL 2.1 or later and
+ * Apache License 2.0. (starting with JNA version 4.0.0).
+ *
+ * You can freely decide which license you want to apply to
+ * the project.
+ *
+ * You may obtain a copy of the LGPL License at:
+ *
+ * http://www.gnu.org/licenses/licenses.html
+ *
+ * A copy is also included in the downloadable source code package
+ * containing JNA, in file "LGPL2.1".
+ *
+ * You may obtain a copy of the Apache License at:
+ *
+ * http://www.apache.org/licenses/
+ *
+ * A copy is also included in the downloadable source code package
+ * containing JNA, in file "AL2.0".
+ */
+package com.sun.jna.platform.win32.COM;
+
+import com.sun.jna.Pointer;
+import com.sun.jna.Structure;
+import com.sun.jna.platform.win32.Variant.VARIANT;
+import com.sun.jna.platform.win32.WTypes.BSTRByReference;
+import com.sun.jna.platform.win32.WinNT.HRESULT;
+import com.sun.jna.ptr.IntByReference;
+import com.sun.jna.ptr.LongByReference;
+
+/**
+ * Implementation for {@link IAccessible } interface
+ *
+ * @author Mo Beigi, me@mobeigi.com
+ */
+public class Accessible extends Dispatch implements IAccessible
+{
+    public static class ByReference extends Accessible implements
+            Structure.ByReference {
+    }
+
+    public Accessible() {
```

Comment:
I'm not sure this no-arg constructor would ever be used.  All the `this._invoke...` methods assume a pointer to a COM object is provided in order to offset the appropriate interface functions.